test/Microsoft.AspNetCore.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -522,10 +522,17 @@ public static IEnumerable<object[]> HtmlHelperLinkGenerationData
         }
 
         [Fact]
-        public async Task BindPropertyCanBeAppliedToControllers()
+        public async Task BindPropertiesAttribute_CanBeAppliedToControllers()
         {
+            // Arrange
+            var formContent = new List<KeyValuePair<string, string>>
```

Comment:
Curious: Why not use a `Dictionary<string, string>`? This looks pretty long-winded as-is.

Same for tests below.

Reply:
Huh, didn't know that worked. Will change it.